### ISSUE TITLE
Missing Loss Function in losses.py

### ISSUE THREAD

**Issue by JuneHou** [Arrow2022jv/MCD#2]
Hi,

I find that in the main_MCD.py, there is an option 
`if not config.train_set == 'train+val' and 'LM' in args.loss_fn:`

I'm wondering what is the argd.loss_fn where there is 'LM' in it? I only see the Plain loss module in the losses.py.

Thanks for the help!
